articles/aks/node-access.md

```diff
@@ -178,7 +178,7 @@ To connect to another node in the cluster, use the `kubectl debug` command. For
 
 ## Use Host Process Container to access Windows node
 
-1. Create `hostprocess.yaml` with the following content and replacing `AKSWINDOWSNODENAME` with the AKS Windows node name.
+1. Create `hostprocess.yaml` with the following content and replacing `AKSWINDOWSNODENAME` with the AKS Windows node name. Note that this specification uses the nanoserver base image - which doesn't have powershell. But because it's running as a host process container, powershell is available from the underlying VM.
```

Comment:
```suggestion
1. Create `hostprocess.yaml` with the following content and replacing `AKSWINDOWSNODENAME` with the AKS Windows node name. This specification uses the nanoserver base image which doesn't have PowerShell. But because it's running as a host process container, PowerShell is available from the underlying VM.
```